### DISCUSSION TITLE
Diplot (Double distribution)

### DISCUSSION THREAD

**Discussion by SimoneTentori** [MadAnalysis/madanalysis5#229] [General]
Goodmorning,

is it possible to plot double distribution in madanalysis (e.g. invariant mass vs pseudo-rapidity)?
If not is it a feature that you are planning to implement?

Many thanks, 
Simone

**Comment by jackaraz** [comment]
Hi @Brognor, Ma5 can generate multiple plots simultaneously and produce the Python code (per plot) for you to modify as you like. So you can take the code and the data from there and use it as you want. We are not planning to implement it at the moment, but if you would like to contribute to Ma5, you are more than welcome to do so.

**Reply by SimoneTentori** [reply]
Dear @jackaraz, thank you so much, this is the method I usually follow, but I thought it could be a nice feature to add, I will try to work on it in the free time and if it will work I will contact you again :)

Simone
